docs/releases.mdx

```diff
@@ -16,21 +16,22 @@ See the link to the corresponding release notes for more information on each rel
-| 5.4         | May 2024                      | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#547765)  |                                                                           |
-| 5.3         | February 2024                 | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#5312303) |                                                                           |
-| 5.2         | October 2023                  | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#527)     |                                                                           |
-| 5.1         | June 2023                     | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#519)     |                                                                           |
-| 5.0         | March 2023                    | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#506)     |                                                                           |
-| 4.5         | February 2023                 | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#451)     |                                                                           |
-| 4.4         | January 2023                  | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#442)     |                                                                           |
-| 4.3         | December 2022                 | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#431)     |                                                                           |
-| 4.2         | November 2022                 | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#421)     |                                                                           |
-| 4.1         | October 2022                  | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#413)     |                                                                           |
-| 4.0         | September 2022                | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#401)     |                                                                           |
+| **Release**                     | **General Availability Date** | **Supported** | **Release Notes**                                       | **Install**                                                                 |
+|---------------------------------|-------------------------------|---------------|---------------------------------------------------------|-----------------------------------------------------------------------------|
+| Sourcegraph 5 Release 6 Patch 1 | August 2024                   | ✅             | [Notes](https://sourcegraph.com/docs/CHANGELOG#56185)   | [Install](https://github.com/sourcegraph/sourcegraph/releases/tag/v5.6.185) |
```

Comment:
Note the new naming convention for the releases starting with Sourcegraph 5 Release 6